pkg/cassandra/config/config.go

```diff
@@ -50,8 +50,8 @@ type Configuration struct {
 
 // Authenticator holds the authentication properties needed to connect to a Cassandra cluster
 type Authenticator struct {
-	Basic BasicAuthenticator `yaml:"basic" mapstructure:",squash"`
-	// TODO: add more auth types
+	Basic                 BasicAuthenticator `yaml:"basic" mapstructure:",squash"`
+	AllowedAuthenticators []string           `yaml:"allowedAuthenticators" mapstructure:"allowedAuthenticators"`
```

Comment:
having an extra field in the struct does not mean users can set it. It needs to be exposed as a CLI flag.

Reply:
but I wanted to ask how can we pass multiple authenticators in cli flag ? since it take a string of authenticators

Reply:
we have many CLI flags that accept lists (e.g. servers for Cassandra)